contents/blog/11-lessons-from-ElevenLabs-hackathon.md

```diff
@@ -0,0 +1,95 @@
+---
+date: 2025-03-03
+title: 11 human lessons from the ElevenLans hackathon
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author:
+  - kevan-gilbert
+featuredImage: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/11labshackathon_fc99e8f485.JPG
+featuredImageType: full
+category: General
+---
+
+PostHog sponsored the [recent hackathon hosted by ElevenLabs](https://elevenlabs.io/blog/announcing-the-winners-of-the-elevenlabs-worldwide-hackathon) – I got to be in the room in San Francisco as it unfolded, and behind-the-scenes during the planning and judging. The two days saw furious sprints of heads-down building, punctuated by kitchen snack breaks and hallway pacing, leading to pre-demo jitters and delirious applause as it all came to an end (while the same flow played out in cities around the world). 
```

Comment:
```suggestion
We sponsored the [recent hackathon hosted by ElevenLabs](https://elevenlabs.io/blog/announcing-the-winners-of-the-elevenlabs-worldwide-hackathon) – and I got to be in the room in San Francisco as it unfolded, and behind-the-scenes during the planning and judging. The two days saw furious sprints of heads-down building, punctuated by kitchen snack breaks and hallway pacing, leading to pre-demo jitters and delirious applause as it all came to an end (while the same flow played out in cities around the world). 
```

Nit, but I just feel this makes it feel a bit more personal - which is an angle I like in this piece